dataprofiler/tests/profilers/utils.py

```diff
@@ -3,6 +3,7 @@
 import random
 
 import numpy as np
+from dataprofiler import settings
```

Comment:
as above with the imports, should be a space between these imports since numpy is a 3rd party lib